regparser/commands/current_version.py

```diff
@@ -14,7 +14,7 @@
 logger = logging.getLogger(__name__)
 
 
-def process_if_needed(cfr_title, cfr_part, year):
+def process_if_needed(cfr_title, cfr_part, year, publication_date):
```

Comment:
Good thinking! I think we'll still need a separate cfr_part param as volumes don't contain that info, but the other three vars can be replaced.